nxcomp/src/Loop.cpp

```diff
@@ -6498,7 +6498,7 @@ void SetTimer(int value)
 
   if (isTimestamp(lastTimer.start))
   {
-    int diffTs = diffTimestamp(lastTimer.start, getTimestamp());
+    diffTs = diffTimestamp(lastTimer.start, getTimestamp());
```

Comment:
Using a global variable is a stupid idea in the first place (unless they needed it for persistency, which doesn't seem likely as the value is overwritten before every check.)

I'd rather use a local variable and rename it.